handlers/authenticating/handler.go

```diff
@@ -113,3 +112,11 @@ func UpdateRole(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("HX-Redirect", c.PathTo("dashboard").String())
 }
+
+func ClearSession(w http.ResponseWriter, r *http.Request, session *sessions.Session) error {
+	delete(session.Values, ctx.UserIDKey)
+	delete(session.Values, ctx.OriginalUserIDKey)
+	delete(session.Values, ctx.OriginalUserRoleKey)
+	delete(session.Values, ctx.UserRoleKey)
+	return session.Save(r, w)
+}
```

Comment:
Moved this function over from handers/errors.go and made public, since they do the same thing now. I figured this file is a better place for that.